lib/usecases/run/GetAllRunsUseCase.js

```diff
@@ -164,6 +170,40 @@ class GetAllRunsUseCase {
                     literal("ROUND(alice_dipole_current * IF(`alice_dipole_polarity` = 'NEGATIVE', -1, 1) / 1000)");
                 filteringQueryBuilder.where(computedColumn).is(aliceDipoleCurrent);
             }
+
+            if (muInelasticInteractionRate) {
+                /**
+                 * The computed mu of inelastic interaction rate
+                 *
+                 * @param {function} literal function to create an object representing a database literal expression
+                 * @return {Object} the object representing the column
+                 */
+                const computedColumn = ({ literal }) =>
+                    literal(`\`Run\`.\`inelastic_interaction_rate_avg\` / (
+                        SUBSTRING_INDEX(SUBSTRING_INDEX(\`lhcFill\`.\`filling_scheme_name\`, '_', 4), '_', -1) 
+                        * ${PhysicalConstant.LHC_REVOLUTION_FREQUENCY_HZ})
+                    `);
```

Comment:
it might be worth using a SQL virtual column?

Reply:
There is reference to another table,  a view would be needed

Reply:
Ah yes indeed I read too quickly